gradle.properties

```diff
@@ -8,14 +8,14 @@ minecraft_version=1.19.3
 
 # Architectury
 # Check for latest at https://maven.architectury.dev/architectury-plugin/architectury-plugin.gradle.plugin
-architectury_plugin_version=3.4-SNAPSHOT
+architectury_plugin_version=3.4.145
 # Check for latest at https://maven.architectury.dev/dev/architectury/loom/dev.architectury.loom.gradle.plugin
-architectury_loom_version=1.0-SNAPSHOT
+architectury_loom_version=1.0.312
```

Comment:
I really don't like this. It is almost recommended by architectury that you always use snapshots. You wouldn't want to update these manually either.. IMO at least.

Reply:
Sounds good